products/pdfmorph.rst

```diff
@@ -0,0 +1,142 @@
+	opening and typing into a terminal window or Windows command
+	prompt. It is recommended that you consult online resources and
+	become somewhat familiar before using PDFmorph.
+
+PDFmorph is a Python software package designed to increase the insight
+researchers can obtain from measured atomic pair distribution functions
+(PDFs) in a model-independent way. The program was designed to help a
+researcher answer the question: "Has my material undergone a phase
+transition between these two measurements?"
+
+PDFmorph makes use of several data manipulation techniques to correct
+for benign effects such as thermal expansion (peak shift) and increased
+thermal motion (peak broadening) or a cahnge in scale due to differences
+in incident flux, for example. PDFmorph will do its best to correct
+for these benign effects before computing and plotting a difference
+curve.
+
+One PDF (typically that collected at higher temperature) is identified
+as the "target" PDF and the second is "morphed" by "stretching" (changing
+the r-axis to simulate a uniform lattice expansion), "smearing"
+(broadening peaks through a uniform convolution to simulate increased
+thermal motion), and "scaling" (self-explanatory). PDFmorph will vary
+amplitude of morphing transformations to obtain the best fit between
+morphed and target PDFs, then plot them along with the difference curve
+plotted below.
+
+There are also a few other morphing transformations in the program.
+
+Finally, we note that PDFmorph should work on other spectra that are
+not PDFs, though it has not been extensively tested beyond the PDF.
+
+.. PDFmorph will soon be included in :doc:`xPDFsuite <xPDFsuite>`, a powerful end-to-end software for X-ray PDF analysis. xPDFsuite is designed for flexible, high throughput PDF analyses and contains many features for interrogating, comparing, and modeling data as well as simply doing data reduction.
+
+.. figure:: ../images/pdfmorph_nacl_ex.png
+	:align: center
+
+	Transformation techniques (scale, smear, and stretch, respectively)
+	utilized by PDFmorph applied sequentially on an NaCl sample
+
+
+Installation
+============
+
+By downloading and using this software, you are agreeing to the conditions
+specified in the :doc:`software license <pdfmorph_license>`.
+
+In order to take automatically take care of software dependency
+downloads, we recomment you install PDFmorph using conda. To do that,
+you will first have to download and install *Anaconda* or *mini-conda*
+from `continuum <https://www.continuum.io>`__.
+
+PDFmorph is available for all operating systems supported by Anaconda,
+namely for 32 and 64-bit Linux, Mac OS X, and 32 and 64-bit Windows.
+
+.. note::
+	We recommend that you run PDFmorph in a Python 3 conda virtual
+	environment. With Anaconda or mini-conda installed on your system,
+	first create the virtual environment by typing ::
+
+		conda create --name=<env_name> python=3
+
+	You can give the environment any name you like but it should
```

Comment:
the python=3 ensures it will have python 3 installed so remove "but it should have some Python version 3 installed (though PDFmorph also works on Python 2.7)."  This is just confusing.

Probably the best is at the top say "PDFmorph will run on python 3 and also python 2.7 or higher, but we recommend.....virtual environment.   Here we give instructions for installing the python 3 version.  With Anaconda...."

Reply:
I would suggest to drop instructions for creating new environment, it is not really needed as pdfmorph is quite lightweight package.

Perhaps the installation instructions should be
```
conda install --channel=diffpy diffpy.pdfmorph
```
or if you use conda-forge
```
conda install --channel=conda-forge diffpy.pdfmorph
```

Reply:
its for novice users who are not familiar with conda.  Maybe start this section out from right below the license information:

"PDFmorph is distribute and is best installed using conda. It should run in most python 3 conda environments and can be directly installed in an existing environment (using the commands listed below).  If you do not currently have conda installed, here are some lightweight instructions for getting started:
 1. download and install *Anaconda* or *mini-conda* from `continuum <https://www.continuum.io>`__."
 1. create a python 3 environment with the command 
```
conda create --name=<env_name> python=3
```
where you can give the environment any name you like.

You only need to create the environment once,
	but every time you want to use PDFmorph you will have to
	reactivate the environment in which it is installed by typing ::

...and so on

Reply:
Simon could you guve the new version a look?